hat/assets/js/apps/Iaso/domains/users/history/UserLogValue.tsx

```diff
@@ -38,17 +58,22 @@ export const UserLogValue: FunctionComponent<Props> = ({ fieldKey, value }) => {
             case 'org_units':
```

Comment:
When you remove a user role, you get a `--` instead in the After column to show that there's nothing. I would do the same with the location I guess 
![image](https://github.com/user-attachments/assets/17ca2ac8-c0a6-4cd7-a89e-16f80daa5856)

Reply:
i would display a placeholder in location too, has this is the general behaviour a bit everywhere